### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by nnsW3**
Hello,
I fixed several minor typos.
Hope it helps.
Br, Elias.

### CODE REVIEW

File: docs/src/confidential-token/deep-dive/encryption.md [hunk 1]
```diff
@@ -256,7 +256,7 @@ struct ConfidentialTransferInstructionData {
 ```
 
 The fields `pending_balance_credit_counter` and
-`maximum_pending_balance_credit_counter` are used to limit amounts that are
+`maximum_pending_balance_credit_counter` is used to limit amounts that are
```
Comment (CriesofCarrots) [inline docs/src/confidential-token/deep-dive/encryption.md:259]: The subject of this verb is "the fields", so "are" is correct. Please revert this change.

File: docs/src/confidential-token/deep-dive/overview.md [hunk 2]
```diff
@@ -238,7 +238,7 @@ zero-knowledge proofs.
   `lower_bound <= x < upper_bound`.
 
   In the confidential extension, we require that a transfer instruction includes
-  a range proof that certifies the following:
+  a range of proof that certifies the following:
```
Comment (CriesofCarrots) [inline docs/src/confidential-token/deep-dive/overview.md:241]: A "range proof" is a specific type of cryptographic proof. Please revert this change.

File: docs/src/confidential-token/quickstart.md [hunk 3]
```diff
@@ -23,11 +23,11 @@ To create a new mint with confidential transfers enabled, run:
 $ spl-token --program-id TokenzQdBNbLqP5VEhdkAS6EPFLC1PHnBqCXEpPxuEb create-token --enable-confidential-transfers auto
 ```
 
-The `auto` keyword means that any token user can permissionlessly configure their
+The `auto` keyword means that any token user can permissionless configure their
```
Comment (CriesofCarrots) [inline docs/src/confidential-token/quickstart.md:26]: The adverb was correct here. Please revert.

File: docs/src/confidential-token/quickstart.md [hunk 4]
```diff
@@ -23,11 +23,11 @@ To create a new mint with confidential transfers enabled, run:
...
 account to perform confidential transfers.
 
 If you would like to gate confidential transfer functionality to certain users,
-you can set the approve policy to `manual`. With this approve policy, all users
+you can set the approved policy to `manual`. With this approved policy, all users
```
Comment (CriesofCarrots) [inline docs/src/confidential-token/quickstart.md:30]: "Approve policy" is a compound phrase. Please revert these changes.

**Review by CriesofCarrots — COMMENTED** [review]
Hi, @nnsW3 , thanks for taking a crack here.
However, quite a few of your proposed changes are incorrect, either grammatically or because we're using compound phrases that refer to specific things. I've made a few comments about these below.

Ultimately, I would approve the changes you've made to `binary-option/README.md`. Can you please edit this PR to just the changes in that file?

**Comment by nnsW3** [comment]
changed them.
Glad I could help.

**Review by CriesofCarrots — APPROVED** [review]
